server/app/api/v1/agents.py

```
@@ -10,17 +10,12 @@
         agents = agent_manager.get_all_agents()
-        return [
-            AgentResponse(agent_id=id, agent_name=name)
-            for id, name in agents
-        ]
+        return [AgentResponse(agent_id=id, agent_name=name) for id, name in agents]
     except Exception as e:
-        raise HTTPException(
-            status_code=500,
-            detail=f"Error fetching agents: {str(e)}"
-        )
+        raise HTTPException(status_code=500, detail=f"Error fetching agents: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)